app/services/post_alerter.rb

```diff
@@ -719,6 +719,13 @@ def email_using_group_smtp_if_configured(post)
       # email to the user who was just added by CC. In this case the OP probably
       # replied and CC'd some people, and they are the only other topic users.
       return if post.incoming_email.cc_addresses_split.include?(to_address)
+
+      # We don't want to create an email storm if someone emails the group and
+      # CC's 50 support addresses from various places, which all then respond
+      # with auto-responders saying they have received our email. Any auto-generated
+      # emails should not propagate notifications to anyone else, not even
+      # the regular topic user notifications.
+      return email_addresses.dup.uniq if post.incoming_email.is_auto_generated?
```

Comment:
I don't have much context about the changes here but I'm wondering how the `.dup.uniq` will prevent the email storm if the problem is that 50 support addresses are in CC.

Reply:
I guess this is a little unclear -- the emails returned here are the ones we skip sending:

https://github.com/discourse/discourse/blob/a9f421bcb1cfff119a570ff9528d7b15f668e218/app/services/post_alerter.rb#L631-L635

I could put them in a different named variable if that helps, like we do at the end of this function:

https://github.com/discourse/discourse/blob/a9f421bcb1cfff119a570ff9528d7b15f668e218/app/services/post_alerter.rb#L743-L745

Reply:
So this is saying, skip emailing everyone if the incoming email was auto generated.

Reply:
O icic that certainly clear things up. Thank you for the explanation :+1: